src/CoreBundle/Resources/translations/admin.de.xliff

```diff
@@ -1118,15 +1132,10 @@
                 <source>chameleon_system_core.cms_module_page_tree.legend_connected_to_protected_page</source>
                 <target>Navigationspunkt mit zugriffsbeschränkter Seite</target>
             </trans-unit>
-            <trans-unit id="chameleon_system_core.cms_module_page_tree.legend_hidden">
```

Comment:
Are these entries renamed (and thus the old name removed)?

Reply:
"chameleon_system_core.cms_module_page_tree.legend_hidden" was unclear. I renamed it to ...legend_node_hidden"

I readded it with a deprecated note.